charts/redpanda/values.schema.json

```diff
@@ -721,12 +709,12 @@
                     "port": {
                       "type": "integer"
                     },
-                    "type": {
-                      "type": "string",
-                      "pattern": "^NodePort$"
-                    },
-                    "nodePort": {
```

Comment:
LoadBalancer Services still expose a NodePort, so I think this `nodePort` label is fine to keep.

From the Kubernetes docs:

>To implement a Service of type: LoadBalancer, Kubernetes typically starts off by making the changes that are equivalent to you requesting a Service of type: NodePort. The cloud-controller-manager component then configures the external load balancer to forward traffic to that assigned node port.